systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java

```diff
@@ -196,6 +210,13 @@ public static void waitForBrokersDrained(TimeoutBudget budget, Address... destin
         TimeMeasuringSystem.stopOperation(operationID);
     }
 
+    public static void waitForForwardersReady(TimeoutBudget budget, Address... destinations) throws Exception {
+        String operationID = TimeMeasuringSystem.startOperation(SystemtestsOperation.ADDRESS_WAIT_FORWARDERS);
+        Thread.sleep(2000);
```

Comment:
I think you should measure address wait after 2s sleep

Reply:
Shouldn't this sleep be removed? I don't understand why its there.

Reply:
yes, I just copied from other methods that also had, I'm going to remove it